### PR TITLE
expr,dataflow: remove mfp dead code

### PR DESCRIPTION
**Pull request by ruchirK**

### CODE REVIEW

File: src/expr/src/linear.rs [hunk 1]
```diff
@@ -1115,7 +1115,7 @@ pub mod plan {
         /// when one wants to capture the resulting datums without packing
         /// and then unpacking a row.
         #[inline(always)]
-        pub fn evaluate_iter<'b, 'a: 'b>(
+        pub fn evaluate_iter2<'b, 'a: 'b>(
```
Comment (ruchirK) [inline src/expr/src/linear.rs:1118]: oops

**Comment by ruchirK** [comment]
@frankmcsherry I was just reading the code to reunderstand how MFPs worked when I noticed that these functions appear to be unused. Not sure if you wanted them around for a specific reason, figured I would throw up a PR just in case, feel free to decline

**Comment by frankmcsherry** [comment]
Looks good to me. I swore I deleted that file in the LIR PR, but I guess not.

**Review by frankmcsherry — APPROVED** [review]
LGTM!

**Pull request merged**